api/src/main/java/org/openmrs/module/attachments/AttachmentsConstants.java

```diff
@@ -69,6 +73,9 @@
 	
 	public static final String UNKNOWN_MIME_TYPE = "application/octet-stream";
 	
+	public static final List<String> EXECUTABLE_MIME_TYPES = Arrays.asList("application/vnd.microsoft.portable-executable",
+	    "text/javascript", "application/javascript", "application/sql", "application/x-sh", "application/java-archive" "application/x-httpd-php","application/x-vbs", "application/x-python-code", "text/vbscript", "text/x-python");
```

Comment:
@isears @ibacher, does it look like a decent shortlist of bad guys?

Reply:
It seems like mimetypes can get pretty crazy, but maybe we should add in `application/x-ms-installer` and `application/x-elf` as well @jnsereko

Reply:
I'd probably drop `application/sql` from the list. Also, a few additions:
* `application/x-applescript`
* `application/x-ruby`
* `application/x-perl`
* `application/wasm`

That should cover many of the more common scripting languages (I can't find any even semi-standard MIME Types for batch and ps1 files).

Reply:
Hello @ibacher i have  removed the `application/sql` mime type  added added more as you suggested.  I guess this catch is good for the start.  
@mks-d @isears @sherrif10 @dkayiwa